coding-intro/coding-intro-core/core-concepts/functions-are-powerful.md

```diff
@@ -0,0 +1,41 @@
+---
+author: kapnobatai136
+
+tags:
+  - discussion
+
+type: normal
+
+category: must-know
+revisionQuestion:
+  formats:
+    - fill-in-the-gap
+  context: standalone
+---
+
+# Functions are powerful
+
+---
+## Content
+
+Functions are one of the most powerful tools that a programmer has.
+
+Imagine we're coding a robot chef 🤖. Making a function for turning on the oven can make our life much easier for several reasons:
+
+- **Reusability**
+  Instead of rewriting all the oven instructions 50 times, we can write one function and just call that function whenever we need
```

Comment:
```suggestion
  Instead of rewriting all the oven instructions 50 times, we can write one function and just call that function whenever we need.
```